rootfs/usr/local/include/toolbox/kops/Makefile.defaults

```diff
@@ -10,13 +9,8 @@ export KOPS_STATE_STORE ?= s3://$(CLUSTER_STATE_BUCKET)
 
 export KOPS_CLOUD ?= aws
 export KOPS_ZONES ?= $(AWS_REGION)a
-
-export KOPS_MASTER_SIZE ?= t2.medium
 export KOPS_MASTER_ZONES ?= $(KOPS_ZONES)
 
-export KOPS_NODE_SIZE ?= t2.medium
```

Comment:
Why are we deleting the defaults here? If we delete them here, we should move them elsewhere. I'm okay with moving them to the `Dockerfile`